src/styles/page-template/custom/code.njk

```diff
@@ -37,34 +37,36 @@ ignoreInSitemap: true
 {% endblock %}
 
 {% block bodyStart %}
-  {{ govukCookieBanner({
-    ariaLabel: "Cookies on [name of service]",
-    messages: [
-      {
-        headingText: "Cookies on [name of service]",
-        html: '<p class="govuk-body">We use some essential cookies to make this service work.</p>
-          <p class="govuk-body">We’d also like to use analytics cookies so we can understand how you use the service and make improvements.</p>',
-        actions: [
-          {
-            text: "Accept analytics cookies",
-            type: "button",
-            name: "cookies",
-            value: "accept"
-          },
-          {
-            text: "Reject analytics cookies",
-            type: "button",
-            name: "cookies",
-            value: "reject"
-          },
-          {
-            text: "View cookies",
-            href: "#"
-          }
-        ]
-      }
-    ]
-  }) }}
+  <form method="post">
```

Comment:
Not sure about this change, as it means clicking the accept / reject buttons in [the example](https://deploy-preview-3116--govuk-design-system-preview.netlify.app/styles/page-template/) results in a `POST` to `/styles/page-template/custom/`.

On the Netlify preview, that results in a 404 with no page content (displays as a white screen in the iframe, and a Chrome error page if opened in a new tab). Not sure what the behaviour would be on PaaS.

I appreciate technically it's needed, but wonder if we should accept that this is only meant to be an illustration of how to customise parts of the page template and omit the `form`?

Reply:
That's a very good point

How about we wire up `preventFormSubmission()` like on example pages?

```suggestion
  <form action="/form-handler" method="post" novalidate>
```

Reply:
✅ Pushed. The submit event is prevented like all the other iframes